clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -908,6 +908,38 @@ float3 lerp(float3, float3, float3);
 _HLSL_BUILTIN_ALIAS(__builtin_hlsl_lerp)
 float4 lerp(float4, float4, float4);
 
+//===----------------------------------------------------------------------===//
```

Comment:
PR title looks like it may be misnamed, since this is also adding the HLSL builtin functions, it is doing more than just adding it to the backend.